### PR TITLE
[TRAFODION-1685]Fix catalog api for getCatalog and getSchema

### PR DESCRIPTION
**Pull request by kevinxu021**

### CODE REVIEW

File: core/conn/jdbc_type2/native/SrvrSmd.cpp [hunk 1]
```diff
@@ -357,6 +357,7 @@ void odbc_SQLSvc_GetSQLCatalogs_sme_(
 			}
 
 			SQLValue = (SQLValue_def *)outputValueList->_buffer;
+			/* //useless and doesn't be included in MXOS server
```
Comment (hegdean) [inline core/conn/jdbc_type2/native/SrvrSmd.cpp]: Can we remove the code in comments?  Is there a reason why we need to keep it?

**Comment by DaveBirdsall** [comment]
Is this change ready to be committed?

**Comment by hegdean** [comment]
We need to make sure we have regression tests added to the test library?   Kevin, did we add test case for this  fix to our test library?

**Comment by kevinxu021** [comment]
Dave, please feel free to commit.
Anu, For QA test case, i'm not sure. For me, have already done the testing.

**Comment by hegdean** [comment]
Would it be possible just add junit test case for this fix.  Thanks

**Pull request merged**